pkg/resource/analyzer/config.go

```diff
@@ -192,6 +192,38 @@ func validatePolicyConfig(schema plugin.AnalyzerPolicyConfigSchema, config map[s
 	return errors, nil
 }
 
+// ValidatePolicyPackConfig validates a policy pack configuration against the specified config schema.
+func ValidatePolicyPackConfig(schemaMap map[string]apitype.PolicyConfigSchema,
+	config map[string]*json.RawMessage) (err error) {
+	for property, schema := range schemaMap {
```

Comment:
Hmm so I was just thinking about this as I do the documentation.. I'm not sure this respects the top level "required" field.. does it? We may need to rethink this logic.

Reply:
Which would also be the case in the service -- and also another good argument for centralizing the logic